### PR TITLE
refactor(rxHideIfUkAccount): FRMW-382-convert to Quark Service.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-382

- [x] Dev LGTM
- [x] Dev LGTM

---
Refactor `rxApp.rxHideIfUkAccount` to Quark Service.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `rxHideIfUkAccount` Quark Service

### Demo Page
screen shot 2016-02-01 at 1 59 25 pm

### API Docs
screen shot 2016-02-01 at 2 00 48 pm

File: src/quarks/rxHideIfUkAccount/scripts/rxHideIfUkAccount.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-angular.module('encore.ui.rxApp')
+angular.module('encore.ui.quarks')
 /**
  * @ngdoc service
- * @name rxApp.service:rxHideIfUkAccount
+ * @name quarks.service:rxHideIfUkAccount
  * @description
- * Check if account number in URL is of the UK origin
+ * Check if an account number in the URL is of UK origin.
```
Comment (CITguy) [inline src/quarks/rxHideIfUkAccount/scripts/rxHideIfUkAccount.js]: Check if ~~an~~ account number in the URL is of UK origin.

**Comment by 100stacks** [comment]
@CITguy, updated with your feedback. :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**